src/client/opamCommands.ml

```diff
@@ -710,6 +710,116 @@ let list ?(force_search=false) cli =
           $no_switch $depexts $vars $repos $owns_file $disjunction $search
           $silent $no_depexts $package_listing cli $pattern_list)
 
+(* TREE *)
+let tree_doc = "Draw the dependency forest of installed packages."
+let tree ?(why=false) cli =
+  let doc = tree_doc in
+  let build_docs = "TREE BUILDING OPTIONS" in
+  let filter_docs = "TREE FILTERING OPTIONS" in
+  let selection_docs = OpamArg.package_selection_section in
+  let display_docs = OpamArg.package_listing_section in
+  let man = [
+    `S Manpage.s_description;
+    `P "This command displays the forest of currently installed \
+        packages as a Unicode/ASCII-art.";
+    `P "Without argument, it draws the dependency forest of packages with no \
+        dependants. With packages arguments, draws the forest of the specified \
+        packages. When non-installed packages are specified in the arguments, \
+        it will try to simulate installing them before drawing the forest.";
```

Comment:
Should we add a word on `--no-switch` here ?

Reply:
As far as `--no-switch` is well documented, I don't think it's necessary.